src/renderer/assets/scss/_helpers.scss

```diff
@@ -0,0 +1,15 @@
+// ==========================================================================
+// HELPERS
+// ==========================================================================
+
+.text-left {
+  text-align: left;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.text-right {
+  text-align: right;
+}
```

Comment:
Because these are global, they are injected into every component's CSS, therefore creating a ton of duplicate rules. Our pattern is to target and style elements using selectors, not global classes.